.github/workflows/build-and-test.yml

```diff
@@ -10,7 +10,11 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [1.13, 1.14]
+        platform: [ubuntu-latest, macos-latest]
```

Comment:
There's nothing non-portable so I don't think we need to test on macOS. (It's pretty slow on GitHub actions... I'd rather only use it if it buys us something.)

If anything, I'd prefer to add older Go versions and stay on Linux. I think this is worth supporting back to Go 1.11 or Go 1.12.

Reply:
Good point. Will drop the macOS part and focus on testing with older Go versions. Also noticed, that I actually didn't change the action to use the Go version from the build matrix :facepalm: PR now updated.